src/scss/components/_list.scss

```diff
@@ -55,29 +52,37 @@
     display: block;
 
     .list__item {
-      display: block;
       width: auto;
       float: none;
+    }
 
-      &::before {
-        margin-right: 2px;
-      }
+    &[class*='col-'] {
+      display: block;
     }
   }
 
   &__item {
-    @include flex-shrink(0);
-    @include flexbox;
-
-    padding: 8px 0;
```

Comment:
@iondrimba removing padding from here we can have a little bug, like the screenshot bellow:

![screen shot 2017-04-27 at 11 39 19 pm](https://cloud.githubusercontent.com/assets/1279783/25512287/f1cae3da-2ba2-11e7-8f01-54f51dfacfbf.png)

This happens when the `list__item` is very small (a real example is using the grid, when viewport comes to this item).